authors/apps/articles/views.py

```diff
@@ -390,3 +401,70 @@ def retrieve(self, request, slug, *args, **kwargs):
         read = len(words) / 279
         return Response({'read_time': round(read)}, status.HTTP_200_OK)
 
+        
+class ShareArticleViaEmailApiView(CreateAPIView):
+
+    def post(self, request, slug):
+        try: 
+            try:
+                article = Article.objects.get(slug=slug)
+            except ObjectDoesNotExist as e:
+                message = {"errors": {
+                    "message": [
+                        "Article not found."
+                    ]}
+                } 
+                return Response(message, status=status.HTTP_404_NOT_FOUND)  
+         
+            email = request.GET['email']    
```

Comment:
We might need to get add a serializer to validate this email before we send a message.